### DISCUSSION TITLE
oob swaps gone wrong

### DISCUSSION THREAD

**Discussion by SamsterZero** [bigskysoftware/htmx#2611] [Q&A]
I have been using HTMX for a project which has 2 forms and on submitting I have to fetch a result set and also an oob swap for other form. The issue is that it takes all input in the other form are placed out side the second form tag.
Is there and solution to it??

**Comment by Telroshan** [comment]
Hello, could you share an example code ? Ideally an easily runnable one such as a JSFiddle or a codepen, so that we can easily investigate on the situation and find out what's going on!
